pkg/repositories/transformers/node_execution.go

```diff
@@ -88,6 +87,14 @@ func CreateNodeExecutionModel(input ToNodeExecutionModelInput) (*models.NodeExec
 		InputURI: input.Request.Event.InputUri,
 	}
 
+	// Start node is special case where the Inputs and Outputs are the same and hence here we copy the Output uri
```

Comment:
thank you for adding this! just to be clear, do we ever send multiple events for a start node? if so, should we update `UpdateNodeExecutionModel` too?

Reply:
Same here. Please review the updated logic which avoids fetching the data from storage if the InputUri is not set 
At propeller side we set the InputUri on start-node to be empty